### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by Yonashenok**
## what I did 
* Make sure no linter error
* Created module directory and files
* used ES6 syntax
* Create a separate file for each functionality
* Display the current date below the navigation bar
* use library Luxon
# note : Hey there i have this linter error it is not in my file I could not figure why it's happening, but I found out it's global bug 🐛 it been here [issues](https://github.com/webhintio/hint/issues/5614)

### CODE REVIEW

File: modules/formHandler.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    title: title.value,
+    author: author.value,
+    id: `${Math.random().toString(36).substr(2, 9)}`,
+  });
+
+  booksContainer.innerHTML = '';
+  createBookList(books);
+  resetForm();
+  saveToLocalStorage(books);
+};
```
Comment (Qoosim) [inline modules/formHandler.js]: - [ ] It is required to ensure that application is working as expected. It should be possible to add and remove books from the list. Below is the screenshot of what I got while trying to add book to the list.

### Your App
addBook

File: modules/timeDate.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const time = document.querySelector('.time-date');
+
+const setTimeDate = () => {
+  const dt = DateTime.now();
+  const localTime = dt.toLocaleString(DateTime.DATETIME_MED);
+  time.textContent = localTime;
+};
+
+export default setTimeDate;
```
Comment (Qoosim) [inline modules/timeDate.js:11]: - [ ] It is required the current date is displayed below the navigation bar. Kindly fix that.

### Your Website
date

### Project Requirement
project-rqt

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Yonashenok, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/formHandler.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    author: author.value,
+    id: `${Math.random().toString(36).substr(2, 9)}`,
+  });
+
+  booksContainer.innerHTML = '';
+  resetForm();
+  saveToLocalStorage(books);
+  createBookList();
+};
+export default formSubmitHandler;
```
Comment (Strangeal) [inline modules/formHandler.js:26]: - [ ]  As suggested by the previous reviewer, kindly ensure that the application is working as expected. It should be possible to add and remove books from the list. Below is the screenshot of what I got while trying to add book to the list.

File: modules/timeDate.js (hunk 2 above)
Comment (Strangeal) [reply modules/timeDate.js:11]: - [ ] As suggested by the previous and also specified in the project requirements, Please display the `date` below the navigation bar. This way users can track their time for adding books to your app to create a good user experience.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @Yonashenok, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/formHandler.js (hunk 3 above)
Comment (Yonashenok) [reply modules/formHandler.js:26]: # time and Date work fine in  my PC

image

##  The page work fine in my pc. could you please check the demo video 


https://github.com/Yonashenok/Awesome-Books-ES6/assets/99332080/a425bed5-0892-4f80-a1e5-1b8015575051

**Comment by medaminedev66** [comment]
Hello @Yonashenok👋,

_This is an additional review as a result of the second opinion request._

I confirm that your web app functions as expected, allowing users to successfully add and remove books. Additionally, the date is correctly displayed below the navbar. Good job! 👏 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by rustam-tolipov — APPROVED** [review]
Hi @Yonashenok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**